### ISSUE TITLE
I cant connect to Zanox API

### ISSUE THREAD

**Issue by OfertasDescontos** [rtrompier/Zanox-Deeplink-API#3]
Hi rtrompier,

Im trying to use your script, but the curl is retrieving blank. Can you help me?

**Comment by henriquelbsouza** [comment]
Same problem here...

**Comment by OfertasDescontos** [comment]
Henrique, segue uma outra solução:

<?

	DEFINE('_CONNECT_ID', 'XXXXX');

	DEFINE('_AD_SPACE' , 'YYYYY');

	$URL = 'http://www.walmart.com.br/';

	$LINK = 'http://toolbox.zanox.com/tools/api/deeplink?connectid=' . _CONNECT_ID . '&url=' . urlencode($URL) . '&adspaceid=' . _AD_SPACE;



	$deeplink = file_get_contents($LINK);

	$deeplinkXML = simplexml_load_string($deeplink);

	echo $deeplinkXML->url;

?>

**Comment by henriquelbsouza** [comment]
@OfertasDescontos opa, assim que chegar em casa vou testar, valeu :)

No caso ele gera o deep link pro $URL direto? é isso?

**Comment by OfertasDescontos** [comment]
Exatamente. O pessoal da zanox tem um arquivo do word que tem esse passo-a-passo. Mas, resumindo tudo, é isso ai! :)

**Comment by henriquelbsouza** [comment]
Entendi, valeu mesmo amigo, estava sofrendo a uns dias aqui já... assim que testar te dou um feedback :+1:

**Comment by henriquelbsouza** [comment]
@OfertasDescontos funcionou perfeitamente para o que eu precisava, muito obrigado :) 

Você está utilizando em algum sistema/plataforma?

**Comment by bactisme** [comment]
I updated the current class to work with the current toolbox :
https://gist.github.com/bactisme/26b9b4907b60323eea03acfe4e3b4dc4

**Comment by rtrompier** [comment]
hi guys, sorry it's been a while since I have not touched this code.
There have certainly been updated side zanox.
It works for you now?
